src/huggingface_hub/repository.py

```diff
@@ -223,23 +223,30 @@ def clone_from(self, repo_url: str, use_auth_token: Union[bool, str, None] = Non
         token = use_auth_token if use_auth_token is not None else self.huggingface_token
         api = HfApi()
 
+        repo_type, namespace, repo_id = repo_type_and_id_from_hf_id(repo_url)
+
+        if namespace is None and token is None:
```

Comment:
This isn't robust to the repositories that live at the root, so I will think a bit more about this.